src/apscheduler/_schedulers/async_.py

```diff
@@ -529,6 +532,29 @@ async def remove_schedule(self, id: str) -> None:
         self._check_initialized()
         await self.data_store.remove_schedules({id})
 
+    async def pause_schedule(self, id: str) -> None:
+        """Pause the specified schedule."""
+        self._check_initialized()
+        await self.data_store.pause_schedules({id})
+
+    async def unpause_schedule(
+        self,
+        id: str,
+        *,
+        resume_from: datetime | Literal["now"] | None = None,
+    ) -> None:
+        """Unpause the specified schedule.
+
+        By default, the schedule will be resumed as if it had never been paused, and all
+        missed runs will be considered misfires. The ``resume_from`` parameter can be
+        used to specify a different time from which to resume the schedule. The string
+        ``'now'`` can be used as shorthand for ``datetime.now(tz=UTC)``. If
+        ``resume_from`` is not ``None``, then the trigger will be repeatedly advanced
+        until the next fire time is at or after the specified time.
```

Comment:
This is missing Sphinx-style parameter documentations (see other methods for examples). Also, the convention, at least in my projects, is to start the docstring text on a new line after the triple quotes, and add an empty line at the end if the docstring is immediately followed by code.

Reply:
Addressed in 61f7051